src/caselawclient/Client.py

```diff
@@ -209,12 +209,14 @@ def get_press_summaries_for_document_uri(
         Returns a list of PressSummary objects associated with a given Document URI
         """
         vars: query_dicts.GetComponentsForDocumentDict = {
-            "parent_uri": DocumentURIString(uri if uri.startswith("/") else "/" + uri),
+            "parent_uri": uri,
             "component": "pressSummary",
         }
         response = self._send_to_eval(vars, "get_components_for_document.xqy")
         uris = get_multipart_strings_from_marklogic_response(response)
-        return [PressSummary(uri.strip(".xml"), self) for uri in uris]
+        return [
+            PressSummary(DocumentURIString(uri.strip("/").strip(".xml")), self) for uri in uris
```

Comment:
Is there scope for a "parse unclean string" method on DocumentURIString

Reply:
Yes; I think it should be a helper function, and `MarkLogicDocumentURIString.as_DocumentURIString()` should basically just call that so we can easily flip between the two.